src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todosLength: number | undefined;
+  selectFilter: Filter;
+  switchFilter: (selector: Filter) => void;
+  activeClearBtn: boolean;
+  clearCompleted: () => void;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todosLength,
+  selectFilter,
+  switchFilter,
+  activeClearBtn,
+  clearCompleted,
+}) => {
+  const changeFilterBy = (filter: Filter) => {
```

Comment:
seems you repeat this logic. You can change the filter state directly, no need to create a handler each time